modules/cudf/src/series.ts

```diff
@@ -648,6 +648,65 @@ export class AbstractSeries<T extends DataType = any> {
     return this.__construct(this._col.gather(selection._col));
   }
 
+  /**
+   * perform nLargest or nSmallest operation
+   * @ignore
+   */
+  _nLargestOrSmallest(ascending: boolean, n: number, keep: DuplicateKeepOption) {
```

Comment:
```suggestion
  protected _nLargestOrSmallest(ascending: boolean, n: number, keep: DuplicateKeepOption) {
```

Reply:
We could also just make this a utility function at the bottom of the file. Then it's not on the `Series` prototype when people look at series instances in the debugger.